Tree/readme.md

```diff
@@ -55,3 +56,4 @@ that, this property should be satisfied at every node in the tree.
 * Spiral Traversal of Binary Tree ----> [C++](/Code/C++/spiral_traversal_of_binary_tree.cpp) 
 * Root to Leaf Path with Given Sum ----> [C++](/Code/C++/Root_to_leaf_path_with_given_sum.cpp)
 * Lowest common ancestor in a binary tree ----> [Python](/Code/Python/LCA_in_binary_tree.py)
+* BFS traversal on Binary Tree ----> [C++](/Code/C++/binary_tree_BFS_traversal.cpp)
```

Comment:
also, please arrange these links in alphabetical order... i.e. 
1. BFS...
2. construction...
3.....
.. Searching
.... Spiral

Reply:
Done sir